dmoj/cptbox/helper.h

```diff
@@ -27,8 +28,16 @@ struct child_config {
     int *seccomp_handlers;
     // 64 cores ought to be enough for anyone.
     unsigned long cpu_affinity_mask;
+    const char **landlock_read_exact_files;
```

Comment:
This may end up looking a little cleaner if we have a

```c
struct {
    const char **read_exact_files;
    ...
} landlock;
```

or even

```c
const struct {
    char **read_exact_files;
    ...
} landlock;
```